Readme game instructions

SunnySparks opened the pull request:
In this pull request we created the instructions for players to use the tic-tac-toe app correctly

SunnySparks committed:
Update README.md

Add instructions for game

SunnySparks committed:
Upload game screenshot

SunnySparks committed:
Update README.md

Add screenshot to readme file

BrittanyBlake reviewed (changes requested):
### Status: Changes Required  🏗️ 🔨 
****
### Hello Team  👋 
You are doing a great job with this project! 🤩 👍 You are almost done! 💪 There are just a few small issues that need to be resolved before you can move on: ⬇️ 
****

- [x] Your README does not explain the rules of the game. 
screenshot of requirements 👇 
![Screen Shot 2020-09-29 at 12 53 50 AM](https://user-images.githubusercontent.com/48804258/94493936-3f40fd00-01ee-11eb-90a4-3a3245f641ec.png)

- [x] I was not able to run the game on **repl.it** following the instructions given in your README. Please ensure that the user is able to play your game on **repl.it** using your instructions. 


****
### Kindly make the required changes and resubmit for a new code review once you are ready 🙏


Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

SunnySparks committed:
Update README.md

Added tic-tac-toe instructions and detailed the instructions for demo link usage

SunnySparks committed:
add game instructions image

atenaiis commented:
thank you for your comments, we have made the changes rested.

BrittanyBlake reviewed (approved):
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****
